config/locales/en.yml

```diff
@@ -284,7 +284,7 @@ en:
         events: "Events"
         comments: "Comments"
         wiki: "Wiki"
-      from_other: "from other"
+      from_other: "From other"
```

Comment:
hey @kuja24, only change in this line is required. please undo other changes. Thanks!